### ISSUE TITLE
Add `fail-fast` to Kontrol Cheatsheet

### ISSUE THREAD

**Issue by palinatolmach** [runtimeverification/gitbook-kontrol#30]
We should add `--fail-fast` option available for `kontrol prove` to Kontrol Cheatsheet. The option stops execution of other branches if a failing node is detected.

**Comment by palinatolmach** [comment]
In addition to it, could you please add `--smt-tactic` too? 
```
  --smt-tactic SMT_TACTIC
                        Z3 tactic to use when checking satisfiability. Example: (check-sat-using smt)
```
TY!

**Comment by reekamaharaj** [comment]
@palinatolmach I went ahead and added them both. 

[Kontrol Cheatsheet](https://docs.runtimeverification.com/kontrol/cheatsheets/kontrol-cheatsheet)

Further note: With the intention of keeping the docs consistent I added the SMT tactic flag like this: 
`--smt-tactic $smt_tactic` 

I used this to match what we have in the docs right now. The cheatsheet has the flags documented like this: `--bmc-depth $n` 

Showing that `$n` is something the user will set. This is slightly different than what is in the terminal when you run `kontrol prove -h` there you will see `--bmc-depth BMC_DEPTH` (@JuanCoRo tell me if this is wrong.)

We might want to discuss which way to use. The docs should match what shows up in the terminal when you use `--help`

Personally the way Juan did it by adding $ makes sense to me. I immediately realized that it was something like a variable that the user would set. It also is what is used on the [kup cheatsheet](https://docs.runtimeverification.com/kontrol/cheatsheets/kup-cheatsheet) page. So its consistent on the docs side at the moment.
